tests/lib/pkgdb.sh

```diff
@@ -637,6 +637,7 @@ pkg_dependencies_ubuntu_classic(){
                 fwupd
                 packagekit
                 qemu-utils
+                shellcheck
```

Comment:
why this change in this PR? is this related at all to the journalctl change? If not I would rather this be part of a different PR or otherwise called out in the PR description as a drive-by change.

Reply:
ah is this needed for the test for journal-state itself? If so I would rather see us install shellcheck for just that test rather than for all systems as that will add time / network resources to every run unnecessarily

Reply:
Well in this case it is going to be preinstalled in the image so we won't need to install it anymore. 
Once this PR is merged, then the spread-images project during the weekend will create a test image with all the dependencies for each system, so it is going to install in the image the shellcheck package. Does it make sense?

Reply:
ah if it's preinstalled in the image that's fine then